sdformat_test_files/models/joint_revolute_two_joints_two_links/joint_revolute_two_joints_two_links.sdf

```diff
@@ -0,0 +1,84 @@
+<?xml version='1.0' encoding='utf-8'?>
+<sdf version="1.7">
+  <model name="joint_revolute_two_joints_two_links">
+    <link name="link_1">
+      <visual name="link_1_visual">
+        <geometry>
+          <box>
+            <size>0.1 0.2 0.4</size>
+          </box>
+        </geometry>
+      </visual>
+      <collision name="link_1_collision">
+        <geometry>
+          <box>
+            <size>0.1 0.2 0.4</size>
+          </box>
+        </geometry>
+      </collision>
+      <inertial>
+        <mass>12.3</mass>
+        <inertia>
+          <ixx>0.205</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.17425</iyy>
+          <iyz>0</iyz>
+          <izz>0.05125</izz>
+        </inertia>
+      </inertial>
+    </link>
+    <link name="link_2">
+      <pose>0.1 0 0.1 0 0 0</pose>
+      <visual name="link_2_visual">
+        <geometry>
+          <box>
+            <size>0.1 0.2 0.3</size>
+          </box>
+        </geometry>
+      </visual>
+      <collision name="link_2_collision">
+        <geometry>
+          <box>
+            <size>0.1 0.2 0.3</size>
+          </box>
+        </geometry>
+      </collision>
+      <inertial>
+        <mass>1.23</mass>
+        <inertia>
+          <ixx>0.013325</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.01025</iyy>
+          <iyz>0</iyz>
+          <izz>0.005125</izz>
+        </inertia>
+      </inertial>
+    </link>
+
+    <joint name="joint_revolute_1" type="revolute">
+      <parent>link_1</parent>
+      <child>link_2</child>
+      <axis>
+        <xyz>1 0 0</xyz>
+        <limit>
+          <lower>-1.5</lower>
+          <upper>1.5</upper>
+        </limit>
+      </axis>
+    </joint>
+    <joint name="joint_revolute_2" type="revolute">
+      <pose>0 0.1 0 0 0 0</pose>
+      <parent>link_1</parent>
+      <child>link_2</child>
+      <axis>
+        <xyz>1 0 0</xyz>
```

Comment:
I think they're rigidly connected because the pose of the joints is different. Neither joint can be rotated because the links are pinned together in two places.

Reply:
Ah, makes sense.